src/components/validations/validation-icon.spec.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ValidationIcon from './validation-icon.component';
+import { InputPresentationContext } from '../../__experimental__/components/input';
```

Comment:
`import 'jest-styled-components';` is missing

Reply:
added ✅